pkg/config/apiserver/audit.go

```diff
@@ -0,0 +1,21 @@
+package apiserver
+
+import (
+	auditV1 "k8s.io/apiserver/pkg/apis/audit/v1"
+
+	configV1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/library-go/pkg/operator/apiserver/audit"
+)
+
+func GetPolicy(forProfile string) (*auditV1.Policy, error) {
+	ac := configV1.Audit{
+		Profile:    configV1.AuditProfileType(forProfile),
+		CustomRules: nil,
+	}
+
+	ap, err := audit.GetAuditPolicy(ac)
+	if err != nil {
+		return nil, err
+	}
+	return ap.DeepCopy(), nil
```

Comment:
Is `DeepCopy` necessary? Looks like `GetAuditPolicy` already copies base policy before editing just to return it.

Reply:
The way I understood the comment on audit.GetAuditPolicy, the prudent thing to do is deepcopy the returned value:

`// Note: the returned policy must not be modifed by the caller prior to a deepcopy.`